php/src/Core/Database/AbstractRepository.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Core\Database;
+
+use App\Core\Entities\AbstractDTO;
+use Cassandra\Rows;
+
+abstract class AbstractRepository
+{
+
+    /** @var string */
+    public $table = '';
+
+    /** @var string */
+    public $primaryKey = '';
+
+    /** @var Connector */
+    public $connection;
+
+    /** @var array */
+    public $keys = [];
+
+    public function __construct(Connector $connector)
+    {
+        $this->connection = $connector;
+    }
+
+    public function getById(string $id): Rows

```

Comment:
here it should return only one line, no?!

Reply:
Actually it returns an ArrayIterator. I still didn't figured out how to proceed with this yet. 

Since the idea is not leave ScyllaDB I think it needs to be like this. What I can improve is to make an Adapter as an Cassandra Package and handle it better. 

But I'll wait until PHP 8.x be available to implement that because I'll rewrite most part of the project.